core/operational-metadata/operational-metadata-jpa/src/main/java/com/thinkbiganalytics/metadata/jpa/feed/OpsFeedManagerFeedProvider.java

```diff
@@ -223,7 +218,12 @@ public void deleteFeedJobs(String category, String feed) {
      * This will call the stored procedure abandon_feed_jobs
      */
     public void abandonFeedJobs(String feed) {
-        repository.abandonFeedJobs(feed);
+
+        String timeStampString = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSS").print(DateTime.now());
+
+        String exitMessage = String.format("Job manually abandoned @ %s", timeStampString);
```

Comment:
After Reviewing the pull request I made some code changes to master to address "[KYLO-423](https://kylo-io.atlassian.net/browse/KYLO-423)"
Can you update/rebase with master and change this method to this:

      `  String exitMessage = String.format("Job manually abandoned @ %s", DateTimeUtil.getNowFormattedWithTimeZone());`